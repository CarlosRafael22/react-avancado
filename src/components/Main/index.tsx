const Main = () => (
  <div>
    <h2>Testing Main</h2>
  </div>
)

export default Main
