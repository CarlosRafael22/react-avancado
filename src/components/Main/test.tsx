import { render, screen } from '@testing-library/react'

import Main from '.'

describe('<Main />', () => {
  it('should render a heading', () => {
    const { container } = render(<Main />)
    expect(
      screen.getByRole('heading', { name: /testing main/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
