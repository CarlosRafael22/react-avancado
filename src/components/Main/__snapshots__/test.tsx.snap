// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> should render a heading 1`] = `
<div>
  <h2>
    Testing Main
  </h2>
</div>
`
